db/schema.rb

```diff
@@ -45,10 +50,32 @@
     t.index ["scheduled_at"], name: "index_good_jobs_on_scheduled_at", where: "(finished_at IS NULL)"
   end
 
-  create_table "schools", force: :cascade do |t|
+  create_table "institution_types", force: :cascade do |t|
     t.string "name"
+    t.integer "rspo_institution_type_id"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["rspo_institution_type_id"], name: "index_institution_types_on_rspo_institution_type_id", unique: true
+  end
+
+  create_table "institutions", force: :cascade do |t|
+    t.bigint "institution_type_id"
+    t.integer "rspo_institution_id"
+    t.integer "rspo_institution_type_id"
```

Comment:
Those probably cannot be null

Reply:
they are set as not null in the current migration that creates them but these changes weren't applied to the schema yet.